out-of-sync-global-dns/workload.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: out-of-sync-global-dns
+  name: out-of-sync-global-dns
+spec:
+  selector:
+    matchLabels:
+      app: out-of-sync-global-dns
+  template:
+    metadata:
+      labels:
+        app: out-of-sync-global-dns
+    spec:
+      containers:
```

Comment:
Need to add `args` to execute the script

Reply:
Fixed